src/SurveyService.Data/OptionRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Data.Provider;
+using LT.DigitalOffice.SurveyService.Models.Db;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.SurveyService.Data;
+
+public class OptionRepository : IOptionRepository
+{
+  private readonly IDataProvider _provider;
+
+  public OptionRepository(
+    IDataProvider provider)
+  {
+    _provider = provider;
+  }
+
+  public async Task<Guid> CreateAsync(DbOption dbOption)
```

Comment:
<Guid?>

Reply:
Done

Reply:
@vorobevamv If DbOption has property Id, of type Guid, why would we return option.Id as Guid?